deployments/jobs.yaml

```diff
@@ -53,3 +53,13 @@ objects:
     appName: content-sources-backend
     jobs:
       - cleanup-missing-domains
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdJobInvocation
+  metadata:
+    labels:
+      app: content-sources-backend
+    name: set-domain-label-2025-03-18
+  spec:
+    appName: content-sources-backend
+    jobs:
+      - set-domain-label
```

Comment:
Lets copy this job to  jobs-stage.yaml and jobs-prod.yaml and then we can update app-interface to run a different stage and prod.  I'd just remove the rest of the jobs in this file for now

Reply:
alrighty, moved it to `jobs-stage` and `jobs-prod` 😄